source/features/cross-deleted-pr-branches.tsx

```diff
@@ -15,17 +15,22 @@ function init(): void {
 	}
 
 	const deletedBranchName = lastBranchAction.textContent!.trim();
-	const repoRoot = select<HTMLAnchorElement>('.head-ref a')!.href.split('/', 5).join('/');
+
+	const repoRoot = select<HTMLAnchorElement>('.head-ref a')!;
```

Comment:
This isn't `repoRoot` but `headRefLink`
```suggestion
	const headRefLink = select<HTMLAnchorElement>('.head-ref a')!;
```